knowledge-base/combine-data-from-multiple-web-services-in-one-table.md

```diff
@@ -0,0 +1,52 @@
+---
+title: How to Combine Data from Multiple Web Service End Points in One Table?
+description: Use multiple web services in the same table with end-point URLs depending on the parent data
+type: how-to
+page_title: 
+slug: combine-data-from-multiple-web-services-in-one-table
+position: 
+tags: 
+ticketid: 1588649
+res_type: kb
+---
+
+## Environment
+<table>
+	<tbody>
+		<tr>
+			<td>Product</td>
+			<td>Progress® Telerik® Reporting</td>
+		</tr>
+	</tbody>
+</table>
+
+
+## Description
+I want to create a report containing a Table/List with multiple countries, or in the general case, other main objects. I should use an endpoint that provides me 
+with all the countries names. 
+I should then get a list of countries data objects with some detail data in it (e.g. "name", "region", "population", etc. for the particular country or other main object).
+The country name, or in the general case another unique field, can be sent as a parameter to another endpoint to retreive the detailed information about the country or the main object. 
+
+The problem is that the data I need in my table is split into these two endpoints - 
+one that gives me a list of the main objects and one that gives me information about one of the objects in the list through a request parameter.
+
+## Solution
+Use nested Tables/[Data Items]({%slug telerikreporting/designing-reports/connecting-to-data/data-items/overview%}) and [Bindings]({%slug telerikreporting/designing-reports/connecting-to-data/expressions/using-expressions/bindings%}) 
+to set the inner table `DataSource.ServiceUrl` property.
```

Comment:
The JsonDataSource component is mentioned below and it would probably be good to mention here that this binding can only work on data items that use a WebServiceDataSource so that there are no users trying it with a JsonDataSource.